prisma/seed.ts

```diff
@@ -33,16 +64,14 @@ const generateFavorites = async () => {
  * @param userId the user we're picking favorites for
  * @param userCount the total amount of users in our database
  * @param favoriteCount the number of favorites each user should have
- * @param seed the seed we're using to generate random ints
  * @returns a list of objects with a single key ``id`` mapping to a int represented as a string
  */
 const pickConnections = (
   userId: number,
   userCount: number,
-  favoriteCount: number,
-  seed: string
+  favoriteCount: number
 ) => {
-  const random = Random.create(seed);
+  const random = Random.create();
```

Comment:
What is the reasoning for getting rid of this? I agree that it isn't too necessary, but I don't see why we wouldn't keep it at this point.